### PR TITLE
Bugsfix

### PR DESCRIPTION
**Pull request by forever-Agriculture**
Add user bug fixed, navigation bar improved, redundant flashes removed, fixed bug with removed user log in, datetime output improved, spelling changes for more consistent code

### CODE REVIEW

File: backend/views/views.py [hunk 1]
```diff
@@ -149,16 +145,16 @@ def login():
     if form.validate_on_submit():
         user = db.session.query(User).filter(
             User.email == form.email.data).first()
-        if user.check_password(form.password.data):
+        if user and not user.delete_date and user.check_password(form.password.data):
             session['user_id'] = user.id
+            session['auth'] = True
```
Comment (serge30) [inline backend/views/views.py]: You don't need separate field to indicate whether user is authenticated, you may just check presence of `user_id` in the session, as it already implemented in `admin_permission` decorator.

File: backend/models/users.py [hunk 2]
```diff
@@ -27,7 +26,7 @@ class User(db.Model):
     name = db.Column(db.Text)
     alias = db.Column(db.Text)
     email = db.Column(db.Text)
-    hashed_password = db.Column(db.Text, nullable=False)
```
Comment (serge30) [inline backend/models/users.py:30]: In what cases this field can be NULL?

**Review by serge30 — CHANGES_REQUESTED** [review]

File: backend/models/users.py (hunk 2 above)
Comment (forever-Agriculture) [reply backend/models/users.py:30]: when we create a new user (in add user option) there's no password field

Comment (forever-Agriculture) [reply backend/models/users.py:30]: so eventually we got an error, this can fix the error until we add a password field or whatever we're gonna do

Comment (serge30) [reply backend/models/users.py:30]: OK, got it. Let it be in such way.

File: backend/views/views.py (hunk 1 above)
Comment (forever-Agriculture) [reply backend/views/views.py]: agree, additional useless code

**Review by serge30 — CHANGES_REQUESTED** [review]
Fix also permissions mode for `backend/static/assets/js/app.js` file

File: backend/models/users.py (hunk 2 above)
Comment (forever-Agriculture) [reply backend/models/users.py:30]: i find it just a temporary feature, cause we need to let users a possibility to create passwords or whatever in future

**Review by serge30 — APPROVED** [review]
Approved.

**Comment by forever-Agriculture** [comment]
Add user bug fixed, navigation bar improved, redundant flashes removed, fixed bug with removed user log in, datetime output improved, spelling changes for more consistent code

**Pull request merged**